programs/cleos/main.cpp

```diff
@@ -448,8 +450,16 @@ fc::variant push_transaction( signed_transaction& trx, const std::vector<public_
 
    packed_transaction::compression_type compression = to_compression_type( tx_compression );
    if (!tx_dont_broadcast) {
+      EOSC_ASSERT( !(tx_use_old_rpc && tx_use_old_send_rpc), "ERROR: --use-old-rpc and --use-old-send-rpc are mutually exclusive" );
+      packed_transaction_v0 pt_v0(trx, compression);
       if (tx_use_old_rpc) {
-         return call(push_txn_func, packed_transaction_v0(trx, compression));
+         EOSC_ASSERT( !tx_read_only, "ERROR: --read-only can not be used with --use-old-rpc" );
+         EOSC_ASSERT( !tx_rtn_failure_trace, "ERROR: --return-failure-trace can not be used with --use-old-rpc" );
+         return call(push_txn_func, pt_v0);
+      } else if (tx_use_old_send_rpc) {
+         EOSC_ASSERT( !tx_read_only, "ERROR: --read-only can not be used with --use-old-send-rpc" );
+         EOSC_ASSERT( !tx_rtn_failure_trace, "ERROR: --return-failure-trace can not be used with --use-old-send-rpc" );
```

Comment:
We want the default on develop-boxed to return failure traces. Need an option to disable returning a failure trace. That may require changing python tests to pass the option as tests may expect no trace on failure.

Reply:
working on that